pkg/action/actioncontroller.go

```diff
@@ -76,10 +75,10 @@ func (ac *ActionController) Reconcile(ctx context.Context, request reconcile.Req
 		return reconcile.Result{RequeueAfter: controllers.DefaultRequeueError}, err
 	}
 
-	if action.Status.Stage == storkv1.ActionStageFinal {
```

Comment:
the presence of this code block was resulting in action resources in Final stage not entering the handle block and hence we were not able to remove the finalizer to be able to delete the action resource. So have commented this block for now. Side effect is that even action resources in Final state will get requeued.

Reply:
Let's remove the commented lines, this is the behaviour in most of our other controllers where they enter into handle() and nothing happens because it's in Final stage.

Reply:
Makes sense. Will remove it during the revision.